### ISSUE TITLE
stamp_changelog.rb:39: undefined method `empty?' for nil:NilClass

### ISSUE THREAD

**Issue by dusek** [pajapro/fastlane-plugin-changelog#5]
I get the above error with running the `stamp_changelog` action in a lane:

```ruby
  desc "Test Change Log maintenance"
  lane :changelog do
    increment_build_number
    changelog = read_changelog
    badge
    # `gym` would be normally here
    clean_build_artifacts
    reset_git_repo(force: true, files: [
        'appname/Assets.xcassets/AppIcon.appiconset/',
    ])
    stamp_changelog(section_identifier: "#{get_version_number} (#{get_build_number})")
    commit_version_bump
  end
```

Indeed [`stamp_changelog.rb:39`](https://github.com/pajapro/fastlane-plugin-changelog/blob/master/lib/fastlane/plugin/changelog/actions/stamp_changelog.rb#L39) contains an `empty?` test.

Stack:
- macOS: 10.11.6
- ruby: 2.0.0p648 + rbenv
- fastlane: 1.97.2
- fastlane-plugin-changelog: 0.2.0

**Comment by alphatroya** [comment]
It looks like `git_tag` argument is mandatory for  `stamp_changelog` command. I pass an empty string to avoid exception but I thing it's not correct behaviour

**Comment by pajapro** [comment]
Thank you @dusek reporting this issue. It is now fixed in [v0.3.0](https://github.com/pajapro/fastlane-plugin-changelog/blob/master/CHANGELOG.md#030---2015-08-09). 

@alphatroya the new fix also means that you do not need to pass an empty string for `git_tag` parameter anymore.
